metals/src/main/scala/scala/meta/internal/metals/scalacli/ScalaCli.scala

```diff
@@ -364,4 +364,34 @@ object ScalaCli {
 
   val name = "ScalaCli"
 
+  private def escape(s: String): String =
+    s.replace("\\", "\\\\")
+  def scalaCliBspJsonContent(args: List[String] = Nil): String = {
```

Comment:
Is this needed only in the tests? Could you move it to a companion objects of BaseScalaCLISuite?

Reply:
Right, it is used only in the test, so I will move this functions to `BaseScalaCLISuite`.